site/_utils/individual.js

```diff
@@ -29,11 +39,32 @@ const filterByLocale = require('../_filters/filter-by-locale');
 const individual = (items, locale) => {
   /** @type PaginatedPage[] */
   let paginated = [];
-  for (const item of items) {
-    if (item.elements?.length > 0) {
-      paginated = paginated.concat(
-        addPagination(filterByLocale(item.elements, locale), item)
-      );
+  for (const item in items) {
+    const authorKey = items[item].key;
+    const authorFeeds = authorsFeeds.find(authorFeeds => {
+      return Object.keys(authorFeeds)[0] === authorKey;
+    });
```

Comment:
That's very costly 🙂  especially since it's nested in a for-loop over all authors.

I see it's coming from external-posts.json being in this structure:

```json
[
  {
    "bramus": [
      {
```

Can't we remove the outer array? Then you could just do `authorFeeds[authorKey]` here.